src/lib/AjaxAdaptorBase.js

```diff
@@ -1,5 +1,5 @@
 export default class AjaxAdaptorBase {
   serverBase() { return '' } // default will use the relative path
   pathRoot() { return '/' }
-  defaultHeaders() { return {} } // any headers that should be in every ajax request
+  headers() { return {} } // any headers that should be in every ajax request
```

Comment:
let's keep this as `defaultHeaders` because in theory you can also munge the headers later on in the actual request.